### PR TITLE
New module: Manage RabbitMQ global parameters [messaging/rabbitmq_global_parameter]

### PR DESCRIPTION
**Pull request by jgkirschbaum**
##### SUMMARY
Contributing rabbitmq_global_parameter.py for managing RabbitMQ global parameters like cluster_name

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
modules/messaging/rabbitmq_global_parameter.py

##### ANSIBLE VERSION
```
ansible 2.2.1.0
```

### CODE REVIEW

File: lib/ansible/modules/messaging/rabbitmq_global_parameter.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        rabbitmq_global_parameter.set()
+        result['changed'] = True
+
+    result['name'] = name
+    result['value'] = value
+    result['state'] = state
+    
+    module.exit_json(**result) 
+
+main()
```
Comment (mattclay) [inline lib/ansible/modules/messaging/rabbitmq_global_parameter.py]: CI failure due to missing conditional when calling main:

```python
if __name__ == '__main__':
    main()
```

Comment (jgkirschbaum) [reply lib/ansible/modules/messaging/rabbitmq_global_parameter.py]: Thank you for your help.

**Comment by samdoran** [comment]
!waffling stale_ci

**Comment by samdoran** [comment]
@jgkirschbaum Are you trying to get CI jobs to rerun by closing/opening the PR?

**Comment by jgkirschbaum** [comment]
@samdoran Yes.

**Review by maxamillion — APPROVED** [review]

**Comment by maxamillion** [comment]
@jgkirschbaum sorry for the lag time on this, thank you for the pull request. If you don't mind updating `version_added` to `2.8` and add integration tests, I'd be happy to merge. Thank you.

**Comment by maxamillion** [comment]
rebuild_merge

**Pull request merged**